meta-mender-core/recipes-mender/mender-snapshot/mender-snapshot.inc

```diff
@@ -0,0 +1,29 @@
+DESCRIPTION = "Mender tool for creating golden-images from a given installation."
+HOMEPAGE = "https://mender.io"
+
+DEPENDS:append = " glib-2.0"
+RDEPENDS:${PN} = "glib-2.0 mender-client (>= 4.0)"
```

Comment:
Hmm, no. I don't think so :thinking: 

We set the `virtual-provider` to `mender-client` no?

Reply:
No, this was removed in 911ea43605a7159aa903dbbf6fbddf6e41e886d2. It didn't work correctly because it was a build-time only provide, and it seemed better to fix it properly by removing it instead of trying to add an additional, ambiguous runtime provide. In Yocto backwards package compatibility is not very important since you won't upgrade one package at a time, like for Debian.

Possibly we might choose a different approach on kirkstone, but let's cross that bridge when we get there, we should stick to the new names in master.

Reply:
Hmm, kk. I see. Why did we decide on doing this breaking change? :thinking: